kornia/augmentation/container/image.py

```diff
@@ -373,3 +371,16 @@ def forward(  # type: ignore
                 param = ParamItem(param.name, None)
             self.update_params(param)
         return self.__packup_output__(input, label)
+
+
+def _get_new_batch_shape(param, batch_shape):
```

Comment:
```suggestion
def _get_new_batch_shape(param, batch_shape: Size) -> Size:
```
`from torch import Size`

Not sure what’s param type

Reply:
ouh I missed the `from torch import Size` :')